product_medical/models/product_template.py

```diff
@@ -7,34 +7,36 @@
 class ProductTemplate(models.Model):
     _inherit = "product.template"
 
-    is_medical = fields.Boolean(string="Is Medical", default=False)
+    is_medical = fields.Boolean(default=False)
 
     medical_certificate_url = fields.Char(string="CE Certificate medical devices")
     medical_class_id = fields.Many2one("medical.class", string="Medical Device Class")
     medicine_category_id = fields.Many2one("medicine.category", string="Drug Category")
-    ppe_category_id = fields.Many2one("ppe.category", string="PPE Category")
+    medical_ppe_category_id = fields.Many2one("ppe.category", string="PPE Category")
 
-    in_vitro_diagnostic = fields.Many2one(
+    medical_in_vitro_diagnostic = fields.Many2one(
         "in.vitro.diagnostic", string="In vitro diagnostics"
     )
-    conformity_declaration_ids = fields.Many2many(
+    medical_conformity_declaration_ids = fields.Many2many(
         "ir.attachment",
-        relation="product_conformity_declaration_rel",
+        relation="product_medical_conformity_declaration_rel",
         column1="product_template_id",
         column2="attachment_id",
         string="Declaration of Conformity",
     )
-    doc_lot_related = fields.Boolean(string="Lot Related", default=False)
-    doc_validity_date = fields.Date(string="Validity Date")
-    ce_certificate_medical_class_ids = fields.Many2many(
+    medical_doc_lot_related = fields.Boolean(string="Lot Related", default=False)
+    medical_doc_validity_date = fields.Date(string="Validity Date")
+    medical_ce_certificate_medical_class_ids = fields.Many2many(
```

Comment:
We could name it `medical_ce_certificate_class_ids` I guess, removing the second `medical` occurrence.